lib/conjur/debify.rb

```diff
@@ -166,11 +168,11 @@ def ignore_file? f
       if perform_deletion
         image = Docker::Image.create 'fromImage' => "alpine:3.3"
         options = {
-          'Cmd'   => [ "sh", "-c", "while true; do sleep 1; done" ],
-          'Image' => image.id,
-          'Binds' => [
-            [ dir, "/src" ].join(':'),
-          ]
+            'Cmd'   => [ "sh", "-c", "while true; do sleep 1; done" ],
+            'Image' => image.id,
+            'Binds' => [
+                [ dir, "/src" ].join(':'),
+            ]
```

Comment:
can you change your indentation to 2 spaces in your IDE? it seems that you auto-indented this.

Reply:
Done